<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrainageChannelFlowEstimatorDialogBase</class>
 <widget class="QDialog" name="DrainageChannelFlowEstimatorDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drainage Channel Flow Estimator</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Program</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="vLayout">
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>239</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>373</height>
          </size>
         </property>
         <property name="title">
          <string>Inputs</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tabTrap">
             <attribute name="title">
              <string>Trapezoidal Channel</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Left Side Slope (width/height ratio)</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="leftSS">
                <property name="minimum">
                 <double>0.010000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.250000000000000</double>
                </property>
                <property name="value">
                 <double>2.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Right Side Slope (width/height ratio)</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QDoubleSpinBox" name="rightSS">
                <property name="minimum">
                 <double>0.010000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.250000000000000</double>
                </property>
                <property name="value">
                 <double>2.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Bottom Width</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Water Depth</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="botWidth">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="minimum">
                 <double>0.010000000000000</double>
                </property>
                <property name="maximum">
                 <double>999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
                <property name="value">
                 <double>5.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <widget class="QDoubleSpinBox" name="depth">
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="minimum">
                 <double>0.010000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.050000000000000</double>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tabSample">
             <attribute name="title">
              <string>Channel from DEM</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>DEM</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cbDEM">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>100</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="btnSampleLine">
                  <property name="text">
                   <string>Draw Cross Section</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Use DEM to estimate slope</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="btnSampleSlope">
                  <property name="text">
                   <string>Draw Long Section (Upstream to Downstream)</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Water Surface Elevation</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="cbWSE">
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-999999.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>999999.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.050000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tabUD">
             <attribute name="title">
              <string>User Defined Channel</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Text File of Station and Elevation Values (space or tab delimited, no header)</string>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="inputFile">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>100</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="focusPolicy">
                   <enum>Qt::WheelFocus</enum>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="btnLoadTXT">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Water Surface Elevation</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="cbUDwse">
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="maximum">
                   <double>999999.989999999990687</double>
                  </property>
                  <property name="singleStep">
                   <double>0.050000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>14</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0" columnstretch="0,0,0,0,0,0">
             <item row="1" column="0" colspan="6">
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0,0">
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Output Directory</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="outputDir">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>100</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="baseSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btnBrowse">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::TabFocus</enum>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="standardButtons">
                  <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="3">
              <widget class="QDoubleSpinBox" name="slope">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.005000000000000</double>
               </property>
               <property name="value">
                <double>0.005000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="label_6">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Manning's Roughness Coefficent</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="indent">
                <number>-1</number>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QDoubleSpinBox" name="n">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.005000000000000</double>
               </property>
               <property name="value">
                <double>0.040000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_10">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Units</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QWidget" name="widget" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <widget class="QRadioButton" name="ft">
                  <property name="focusPolicy">
                   <enum>Qt::TabFocus</enum>
                  </property>
                  <property name="text">
                   <string>ft</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QRadioButton" name="m">
                  <property name="focusPolicy">
                   <enum>Qt::TabFocus</enum>
                  </property>
                  <property name="text">
                   <string>m</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_5">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Slope of Channel</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Documentation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="autoFormatting">
          <set>QTextEdit::AutoNone</set>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
         <property name="source">
          <url>
           <string>qrc:/plugins/FlowEstimator/documentation.html</string>
          </url>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>leftSS</tabstop>
  <tabstop>rightSS</tabstop>
  <tabstop>botWidth</tabstop>
  <tabstop>depth</tabstop>
  <tabstop>cbDEM</tabstop>
  <tabstop>btnSampleLine</tabstop>
  <tabstop>btnSampleSlope</tabstop>
  <tabstop>cbWSE</tabstop>
  <tabstop>inputFile</tabstop>
  <tabstop>btnLoadTXT</tabstop>
  <tabstop>cbUDwse</tabstop>
  <tabstop>ft</tabstop>
  <tabstop>m</tabstop>
  <tabstop>slope</tabstop>
  <tabstop>n</tabstop>
  <tabstop>outputDir</tabstop>
  <tabstop>btnBrowse</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget_2</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DrainageChannelFlowEstimatorDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DrainageChannelFlowEstimatorDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
